### DISCUSSION TITLE
[Error] ResizeObserver loop completed with undelivered notifications

### DISCUSSION THREAD

**Discussion by VadymBezsmertnyi** [davidjerleke/embla-carousel#489] [Help]
Hello.
In the Safari 16.4 (18615.1.26.110.1) browser console, when using embla-carousel 8.0.0-rc02 to 8.0.0-rc05, the following error occurs: [Error] ResizeObserver loop completed with undelivered notifications.
However, when using embla-carousel 7.1.0, there are no errors. Is it possible to fix this myself, or should I wait for a fix from you?

**Comment by davidjerleke** [comment]
@VadymBezsmertnyi thanks for sharing this. Maybe you have more information to add to the bug regarding this console error here:

- #488

Like why this is happening and how to solve it?

Closing this because there’s already a bug report for it.

Best,
David
